### PR TITLE
add: stardust-armillary

### PR DESCRIPTION
**Pull request by Owen-sz**

### CODE REVIEW

File: anda/stardust/armillary/stardust-armillary.spec [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# Exclude input files from mangling
+%global __brp_mangle_shebangs_exclude_from ^/usr/src/.*$
+
+Name:           stardust-armillary
+Version:        %commit_date.%shortcommit
+Release:        1%?dist
+Summary:        Model viewer for Stardust XR.
+URL:            https://github.com/StardustXR/armillary
+Source0:        https://github.com/StardustXR/armillary/archive/%commit/armillary-%commit.tar.gz
```
Comment (madonuko) [inline anda/stardust/armillary/stardust-armillary.spec]: again, don't repeat yourself

File: anda/stardust/armillary/stardust-armillary.spec [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+License:        MIT
+BuildRequires:  cargo cmake anda-srpm-macros cargo-rpm-macros mold
+Requires:       libgcc glibc
```
Comment (madonuko) [inline anda/stardust/armillary/stardust-armillary.spec]: probably don't need this too

File: anda/stardust/armillary/stardust-armillary.spec [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+%description
+A model viewer for Stardust XR which works great for hand tracking, pointers, and controllers.
+
+%prep
+%autosetup -n armillary-%commit
+%cargo_prep_online
+
+%build
+%cargo_build
```
Comment (madonuko) [inline anda/stardust/armillary/stardust-armillary.spec]: remove this

File: anda/stardust/armillary/update.rhai [hunk 4]
```diff
@@ -0,0 +1,7 @@
+if filters.contains("nightly") {
```
Comment (madonuko) [inline anda/stardust/armillary/update.rhai]: don't need this

**Review by madonuko — CHANGES_REQUESTED** [review]
also make yourself the `Packager:` in all 3 packages

File: anda/stardust/armillary/stardust-armillary.spec (hunk 2 above)
Comment (madonuko) [reply anda/stardust/armillary/stardust-armillary.spec]: ?

File: anda/stardust/armillary/update.rhai (hunk 4 above)
Comment (madonuko) [reply anda/stardust/armillary/update.rhai]: ?

**Review by madonuko — CHANGES_REQUESTED** [review]

**Review by madonuko — DISMISSED** [review]

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2087)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2088)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2089)|
|✅|el9|[](https://github.com/terrapkg/packages/pull/2090)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/10943048566) for details

**Pull request merged**